remove_win32_bit/jep.md

```diff
@@ -1 +1,75 @@
-TODO
+# JEP NNN: Removal of Windows 32-bit Support
+
+_Author_	Bruno Borges & George Adams
+_Owner_ Bruno Borges
+_Type_	Feature
+_Scope_	Implementation
+_Status_	Draft
+_Release_	23
+_Component_	hotspot / other
+_Discussion_	jdk dash dev at openjdk dot org
```

Comment:
Maybe porters-dev and build-dev as well if this field takes multiple lists?

Reply:
I don't think it does. We can check when we submit for real.